platform/sun50i/devices.c

```diff
@@ -54,3 +67,13 @@ static struct device r_intc = {
 	.drv     = &sun4i_intc_driver,
 	.drvdata = SUN4I_INTC_DRVDATA { { 0 } },
 };
+
+static struct device r_pio = {
+	.name     = "r_pio",
+	.clock    = CCU_GATE(R_CCU_GATE_R_PIO),
+	.clockdev = &r_ccu,
+	.drv      = &sunxi_pio_driver,
+	.drvdata  = BIT(0), /*< Physically implemented R_PIO ports (0). */
```

Comment:
Update this comment to match what you decide to do with the other one above.

Reply:
Done.